activity_browser/app/controller.py

```diff
@@ -85,10 +87,11 @@ def switch_brightway2_dir_path(self, dirpath):
                 os.path.join(bw.projects._base_data_dir, "projects.db"),
                 [ProjectDataset]
             )
-            bw.projects.set_current(self.get_default_project_name())
-            signals.projects_changed.emit()
+            print('Loaded brightway2 data directory: {}'.format(bw.projects._base_data_dir))
+
+            self.change_project(self.get_default_project_name())
```

Comment:
This change introduces the bug that you mention in your first comment. `change_project` is not aware that the projects db has changed, ie it will do nothing if we switch from `default` in one projects-db to `default` with the projects-db on the new path.